package ch08;

public class Exception11 {
    public static void main(String[] args) {
//        throw new RuntimeException클래스들에();        // RuntimeException 고의로 발생
    }
}
